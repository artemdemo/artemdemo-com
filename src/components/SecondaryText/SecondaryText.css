.secondary-text {
  font-size: 85%;
  margin-bottom: 10px;
  opacity: 0.5;
  transition: 300ms;
}
.secondary-text:hover {
  opacity: 1;
}
