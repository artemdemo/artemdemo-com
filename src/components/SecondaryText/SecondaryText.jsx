import styled from 'styled-components';

const SecondaryText = styled.div`
    font-size: 85%;
    margin-bottom: 10px;
    opacity: 0.5;
    transition: 300ms;
    
    &:hover {
        opacity: 1;
    }
`;

export default SecondaryText;