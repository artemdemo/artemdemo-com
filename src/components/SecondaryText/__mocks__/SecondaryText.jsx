import React from 'react';

const SecondaryText = props => (
    <div data-mock='SecondaryText'>
        {props.children}
    </div>
);

export default SecondaryText;