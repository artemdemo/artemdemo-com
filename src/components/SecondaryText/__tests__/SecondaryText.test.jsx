import React from 'react';
import renderer from 'react-test-renderer';
import SecondaryText from '../SecondaryText';

describe('SecondaryText', () => {
    it('should render', () => {
        const tree = renderer.create(
            <SecondaryText>
                Some text
            </SecondaryText>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});