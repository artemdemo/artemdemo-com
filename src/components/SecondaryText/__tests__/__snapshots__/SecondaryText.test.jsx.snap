// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryText should render 1`] = `
<div
  className="sc-bdVaJa hUnbxe"
>
  Some text
</div>
`;
